mm2src/coins/nft/storage/wasm/wasm_storage.rs

```diff
@@ -315,6 +369,60 @@ impl NftListStorageOps for IndexedDbNftStorage {
             .await?;
         Ok(())
     }
```

Comment:
I think you can redeclare a new chain var and make the conversion to string once and pass by reference instead of doing it every time 
```rust
let chain_str = chain.to_string()
```